src/objects-and-functions/homework-2.js

```diff
@@ -0,0 +1,84 @@
+// --- TASK 1 --- //
+// Задача про обчислення різниці часу
+// Напишіть функцію яка буде буде приймати 3 параметри
+// — початкову дату (string)
+// — кінцеву дату (string)
+// — розмірність ('days', 'hours', 'minutes', seconds)
+// Функція повертатиме часовий період між цими датами згідно розмірності.
+// Також вкажіть значення по замовчуванню для всіх цих параметрів (на ваш вибір).
+// Функція має коректно працювати навіть якщо початкова дата пізніше ніж кінцева дата.
+
+const durationBetweenDates = (dateFrom, dateTo) => {
+    const start = new Date(dateFrom);
+    const end = new Date(dateTo);
+
+    const timeDiff = Math.abs(end - start);
+    const durationInDays = Math.ceil(timeDiff / (1000 * 60 * 60 * 24));
+
+    return `${durationInDays} days`;
+};
+
+// Test of durationBetweenDates():
+// console.log(durationBetweenDates('31 Jan 2022', '03 Feb 2021', 'days')); // поверне '362 days'
+// console.log(durationBetweenDates('01 Jan 2014', '03 Feb 2023', 'days')); // поверне '3320 days'
+// console.log(durationBetweenDates('01 Jan 2018', '03 Feb 2023', 'days')); // поверне '1859 days'
+
+
+
+
+
+// --- TASK 2 --- //
+// Задача про перетворення об'єкту
+// Допустимо у вас є об'єкт, у якому кожен ключ - це назва товару (одним словом), а значення - його ціна.
+// Напишіть функцію яка буде всі ключі переводити у нижній регістр, а всі ціни буде заокруглювати до двох знаків після коми.
+
+const priceData = {
+    Apples: '23.4',
+    BANANAS: '48',
+    oRAngGEs: '48.7584'
+};
+
+const priceData2 = {
+    SnEakerS_AirForce_320: '599.45',
+    SnEakerS_RunnING_110: '428.1',
+    snEakerS_runnING_150: '499.7584'
+};
+
+const optimizer = (data) => {
+    const updatedData = {};
+
+    for (const item in data) {
+        const lowerCaseKey = item.toLowerCase();
+        const roundedPrice = parseFloat(data[item]).toFixed(2);
```

Comment:
Змінна `roundedPrice` не потрібна, можна зразу: 
```suggestion
        updatedData[lowerCaseKey] = parseFloat(data[item]).toFixed(2);
```

Reply:
Таким чином, ми зменшили код на одну строку. Зрозумів. Дякую!

Reply:
Справа не в зменшенні написаного коду, а в мінус одній, не потрібній більше ніде, змінній